### DISCUSSION TITLE
File upload using withMultiPartFormData is not working

### DISCUSSION THREAD

**Discussion by spmathan** [pactumjs/pactum#296] [Q&A]
Recently I upgraded from Pactum **3.1.4** to **3.4.1** version

I am using **withMultiPartFormData** for handling **Uploading CSV file** scenarios since the update seems not working. File is not picked by function. 

when I downgraded to 3.1.4 it works

If anyone else is facing a similar problem. Help me with some workaround

**Comment by ASaiAnudeep** [comment]
A modification related to the form-data was introduced in release [v3.2.0](https://github.com/pactumjs/pactum/releases/tag/v3.2.0) This could potentially be the cause of the problem. But we haven't faced any issues with uploading files.

**Reply by gr00nd** [reply]
[www.pax4pro.com]()

**Reply by spmathan** [reply]
I am using Graphql query with formdata for upload. doesnt seem to be working.

**Reply by ASaiAnudeep** [reply]
Could you kindly provide me with the code you're currently using? This will allow me to better understand the issue you're facing.
